core/server/master/src/main/java/alluxio/master/file/meta/MutableInodeDirectory.java

```diff
@@ -113,8 +113,6 @@ public MutableInodeDirectory setDefaultACL(DefaultAccessControlList acl) {
   /**
    * Generates client file info for a folder.
    *
-   * xAttr is intentionally left out as the information is not yet exposed to clients
-   *
```

Comment:
I decided to expose `xAttr` in `generateClientFileInfo()` since Alluxio gRPCs are now using the field, but this can be undone if we still wish to keep it hidden.